sample_data/templates/NRFA_SITES.yaml

```diff
@@ -76,6 +79,35 @@ one_offs:
       "<iop:hasObjectOfInterest>": <::Catchment>
       "<iop:hasProperty>": <::AreaProperty>
 
+  - name: IdentifierProperty
+    properties:
+      "@id": "<http://fdri.ceh.ac.uk/ref/common/property/identifier>"
+      "@type": "<iop:Property>"
+      "<skos:prefLabel>": "Identifier@en"
+      "<rdfs:label>": "Identifier@en"
+      "^<iop:hasProperty>": "<::COP>"
+      "<skos:inScheme>": "<::PropertyScheme>"
+      "^<skos:hasTopConcept>": "<::PropertyScheme>"
+
+  - name: Catchment
```

Comment:
Name doesn't matter but might be clearer if this was something like "EnvronmentalMonitoringFacility"?